karma.conf.js

```diff
@@ -106,7 +110,7 @@ module.exports = function(config) {
 				`local${require('./package.json').version}`,
 			connectLocationForSERelay: 'localhost',
 			connectPortForSERelay: 4445,
-			startConnect: false
+			startConnect: true
```

Comment:
just to check: this doesn't need to be conditional because we disable saucelabs itself in cases where we don't want it to run, so the proxy never starts?

Reply:
I put it to `true` just for testing it's `!!sauceLabs` now